site/docs/v1/tech/apis/identity-providers/_oauth-idp-response-body.adoc

```diff
@@ -77,6 +82,14 @@ ifeval::["{idp_type}" == "Steam"]
 The top-level web API key to use with the {idp_display_name} Identity Provider when retrieving the player summary info. This value is retrieved from the {idp_display_name} developer website when you setup your {idp_display_name} developer account.
 endif::[]
 
+ifeval::["{idp_type}" == "Nintendo"]
+[field]#identityProvider.uniqueIdClaim# [type]#[String]#::
```

Comment:
Looks like we are missing the `uniqueIdClaim` and `usernameClaim` claim for OpenID Connect, can you add them?

Reply:
yep

Reply:
should be fixed